tests/sample/test_sample_models.py

```diff
@@ -0,0 +1,149 @@
+from datetime import datetime
+import uuid
+
+from dataservice.extensions import db
+from dataservice.api.participant.models import Participant
+from dataservice.api.sample.models import Sample
+from tests.utils import FlaskTestCase
+
+from sqlalchemy.exc import IntegrityError
+
+class ModelTest(FlaskTestCase):
+    """
+    Test database model
+    """
+    def create_participant_sample(self):
+        participant_id ="Test_Subject_0"
+        sample_id ="Test_Sample_0"
+        data={
+        'external_id': sample_id,
+        'tissue_type': 'Normal',
+        'composition': 'Test_comp_0',
+        }
+        sample_0 = Sample(**data)
+        participant_0 = Participant(external_id = participant_id,sample =sample_0)
```

Comment:
Style nit: no spaces around kwarg assignments, spaces after the comma in arguments.